src/plugins/synchronizedFolder/SynchronizedObjectsProvider.js

```diff
@@ -0,0 +1,83 @@
+export const PLANS_FOLDER_KEY = 'folder_synchronized_couchdb';
+const COUCHDB_PERSISTENCE_SPACE = 'mct';
+const MY_ITEMS_LOCATION = 'mine';
+
+export class SynchronizedObjectsProvider {
```

Comment:
See general comments, I suspect we can do away with this and simplify things a bit.

Reply:
Removed